chains/ethereum/config.go

```diff
@@ -180,6 +184,24 @@ func parseChainConfig(chainCfg *core.ChainConfig) (*Config, error) {
 		delete(chainCfg.Opts, EGSSpeed)
 	}
 
+	if itxConfig, ok := chainCfg.Opts[ItxEndpoint]; ok && itxConfig != "" {
+		config.itxEndpoint = &itxConfig
+		delete(chainCfg.Opts, ItxEndpoint)
+	} else {
+		config.itxEndpoint = nil
```

Comment:
This is not strictly necessary since this struct property will already be `nil`, because it isn't specified where the struct is or set anywhere else. If this code ever evolves such that the config struct is configured before this point with values from places other than the chainCfg, this could create a surprise by nilling out that default while chainCfg isn't specifying an itxEndpoint.